privacy.md

```diff
@@ -177,8 +177,8 @@ rights, subject to some limitations, against the Foreman Project:
 * The right to lodge a complaint with your local data protection authority
 
 If you would like to exercise any of these rights, you may do so by contacting
-the Foreman Project at [contact us](mailto:privacy AT community.theforeman.org)
```

Comment:
I think the `AT` part was mostly there to avoid spammers from harvesting it. Even though it's really easy to avoid, it's a common thing to do.

Reply:
Yes, that was probably the original idea. However, the email address was there in plaintext on the same page. So no need to "obfuscate" it in this line. See line 197.

Reply:
Fair enough.